__tests__/service/onesignal/OneSignalUtil.test.ts

```diff
@@ -7,7 +7,8 @@ const mockSubscribeableTopicResponse = {
   code: 200,
   data: {
     topics: [{name: 'android'}, {name: 'apple'}],
-    history: [{name: 'android'}, {name: 'apple'}, {name: 'women'}]
+    history: [{name: 'android'}, {name: 'apple'}, {name: 'women'}],
+    hasUserPosted: true
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `hasUserPosted` to the data object seems fine. It doesn't appear to break any existing functionality and is a simple addition of a new field.

```diff
+    hasUserPosted: true
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->